supplier_product/tests.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from supplier_product.models import SupplierProduct
+
+
+@pytest.mark.django_db()
+class TestSupProduct:
+
+    def test_save_supplier_product(self, supplier_product0):
+        assert supplier_product0 not in SupplierProduct.objects.all()
+        SupplierProduct.save_sup_product(supplier_product0)
+        assert supplier_product0 in SupplierProduct.objects.all()
+
+    def test_delete_supplier_product(self, supplier_product0):
+        assert supplier_product0 not in SupplierProduct.objects.all()
+        SupplierProduct.save_sup_product(supplier_product0)
+        assert supplier_product0 in SupplierProduct.objects.all()
+        supplier_product0.delete()
+        assert supplier_product0 not in SupplierProduct.objects.all()
+
+    def test_increase_supp_product_quantiy(self, supplier_product0):
+        new_quantity = supplier_product0.quantity + 1
+        old_quantity = supplier_product0.quantity
+        supplier_product0.increase_sup_product_quantity(new_quantity)
+        assert supplier_product0.quantity == old_quantity + new_quantity
+
+    def test_decrease_supp_product_quantiy(self, supplier_product0):
+        new_quantity = supplier_product0.quantity - 1
+        old_quantity = supplier_product0.quantity
+        supplier_product0.decrease_sup_product_quantity(new_quantity)
+        assert supplier_product0.quantity == old_quantity - new_quantity
+
+    def test_decrease_supp_product_quantiy_delete(self, supplier_product0):
+        new_quantity = supplier_product0.quantity + 1
+        supplier_product0.decrease_sup_product_quantity(new_quantity)
+        assert supplier_product0 not in SupplierProduct.objects.all()
```

Comment:
didn't you change this method? I think you should search for an error, not for a deletion (and the name of the test should reflect this, like `test_<method_name>_raises_when_no_quantity_left`